swarm/storage/lnetstore.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package storage
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/ethereum/go-ethereum/swarm/chunk"
+	"github.com/ethereum/go-ethereum/swarm/network/timeouts"
+)
+
+type LNetStore struct {
```

Comment:
Read the description of the PR. I agree this is a crap name, but I don't have a better one yet. Any suggestions?

Reply:
The PR description says 

> Added LNetStore (we probably need a better name) - a NetStore wrapper needed for the LazyChunkReader

That is very high level and not saying much. That is why I think Godoc should be added to this object. I can't suggest a better name because I don't understand why it is needed at all?